modules/enabling-custom-images.adoc

```diff
@@ -3,16 +3,16 @@
 [id='enabling-custom-images_{context}']
 = Enabling custom images in {productname-short}
 
-All {productname-short} admin users can import custom workbench images, by default, by selecting the *Settings* -> *Notebook images* navigation option in the {productname-short} dashboard.
+All {productname-short} administrators can import custom workbench images, by default, by selecting the *Settings* -> *Notebook images* navigation option in the {productname-short} dashboard.
 
 If the *Settings* -> *Notebook images* option is not available, check the following settings, depending on which navigation element does not appear in the dashboard:
 
 * The *Settings* menu does not appear in the {productname-short} navigation bar
 +
-The visibility of the {productname-short} dashboard's Settings menu is determined by your user permissions. By default, the **Settings* menu is available to {productname-short} administration users (users that are members of the `rhoai-admins` group). Users with the OpenShift `cluster-admin` role, are automatically added to the `rhoai-admins` group and are granted administrator access in {productname-short}. 
+The visibility of the {productname-short} dashboard's Settings menu is determined by your user permissions. By default, the *Settings* menu is available to {productname-short} administration users (users that are members of the `rhoai-admins` group). Users with the OpenShift `cluster-admin` role are automatically added to the `rhoai-admins` group and are granted administrator access in {productname-short}. 
```

Comment:
In line 12, the first occurrence of the "Settings menu" is not formatted, but the second occurrence is. Also, the occurrence of "dashboard's" can be changed to just "dashboard".